### PR TITLE
Revert "Fixed bug in NeoSharp.Core/Blockchain/Repositories/BlockRepository"

### PR DESCRIPTION
**Pull request by gubanotorious**
Reverts CityOfZion/neo-sharp#460

### CODE REVIEW

File: src/NeoSharp.Application/Controllers/PromptWalletController.cs [hunk 1]
```diff
@@ -124,19 +123,17 @@ public void AccountDeleteCommand(string address)
...
                     var walletPassword = _consoleHandler.ReadPassword();
-                    var accountPrivateKey = _walletManager.DecryptNep2(walletAccount.Key, walletPassword);
+                    byte[] accountPrivateKey = _walletManager.DecryptNep2(walletAccount.Key, walletPassword);
                     var newKeyPassword = _consoleHandler.ReadPassword("\nNew key password:");
                     var newKeyPasswordConfirmation = _consoleHandler.ReadPassword("\nConfirm your password:");
-                    
                     if (newKeyPassword.ToByteArray().SequenceEqual(newKeyPasswordConfirmation.ToByteArray()))
                     {
-                        var nep2Key = _walletManager.EncryptNep2(accountPrivateKey, newKeyPassword);
+                        string nep2Key = _walletManager.EncryptNep2(accountPrivateKey, newKeyPassword);
```
Comment (shargon) [inline src/NeoSharp.Application/Controllers/PromptWalletController.cs:136]: Why this changes?

**Comment by gubanotorious** [comment]
I was just reverting a bad commit / merge...  I didn't actually touch that / intend to if it got changed.
